app/unit-tests/src/org/commcare/android/tests/processing/QuarantinedFormsTest.java

```diff
@@ -0,0 +1,31 @@
+package org.commcare.android.tests.processing;
+
+import org.commcare.utils.FormUploadUtil;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+
+/**
+ * Created by amstone326 on 6/14/17.
+ */
+
+public class QuarantinedFormsTest {
```

Comment:
I took a stab at writing a robolectric test for this, with a mocked `ProcessAndSendTask` object, etc., but it was a mess and didn't seem like it was super valuable based on the amount of stuff I needed to fake, so I am planning to write a Calabash test instead, once the end-to-end work is done